.github/workflows/ci.yml

```diff
@@ -24,7 +25,7 @@ jobs:
       fail-fast: false
       matrix:
         NXF_VER:
-          - "23.04.0"
+          - "24.04.2"
```

Comment:
This and the following lines:
According to https://github.com/nf-core/ampliseq/pull/785/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f, which might or might not be correct, prifole is now
```
        profile:
          - "conda"
          - "docker"
          - "singularity"
```
and test configs under the headline `test_name`.
And those are executed in line 85 with `nf-test test --tag ${{ matrix.test_name }} --profile ${{ matrix.test_name }},${{ matrix.profile }} --junitxml=test.xml`.
As it is now, it seems to me, at least you ignore conda & apptainer.

Reply:
Thanks for looking into that! I needed to change some more things within the ci.yaml 👍